ilc/common/utils.js

```diff
@@ -19,7 +21,12 @@ function makeAppId(appName, slotName) {
     return `${appName.replace('@portal/', '')}__at__${slotName}`;
 }
 
+function clone(source) {
```

Comment:
```suggestion
function cloneObject(source) {
```

Reply:
this method respect not only objects, so I will rename it to `cloneDeep`